Ninano/Ninano/CalendarCollectionViewCell.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  CalendarCollectionViewCell.swift
+//  Ninano
+//
+//  Created by Yoonjae on 2022/07/17.
+//
+
+import UIKit
+
+class CalendarCollectionViewCell: UICollectionViewCell {
```

Comment:
더 이상 상속되지 않는 것을 알기 쉽도록 final이 있으면 좋을듯합니당 
```suggestion
final class CalendarCollectionViewCell: UICollectionViewCell {
```

Reply:
넵 감사합니다!! 지금 수정했습니다.